src/recipe-list.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import GridList from '@material-ui/core/GridList'
+import GridListTile from '@material-ui/core/GridListTile'
+import GridListTileBar from '@material-ui/core/GridListTileBar'
+import { withStyles } from '@material-ui/core/styles'
+import Typography from '@material-ui/core/Typography'
+
+const styles = {
+  title: {
+    marginTop: 150,
+    marginBottom: 80
+  },
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'space-around',
+    overflow: 'hidden'
+  },
+  gridList: {
```

Comment:
Let your `GridList` be nice and big. Those pictures are awesome so the grid should take up a lot of room.